src/client/Microsoft.Identity.Client.Desktop/Microsoft.Identity.Client.Desktop.csproj

```diff
@@ -56,6 +56,20 @@
      <Reference Include="System.Windows.Forms" />
 
    </ItemGroup>
+   <ItemGroup>
+     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.2">
+       <PrivateAssets>all</PrivateAssets>
+       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+     </PackageReference>
+     <PackageReference Include="Microsoft.Internal.Analyzers" Version="2.6.9">
```

Comment:
Try to use floating versions like 

![image](https://user-images.githubusercontent.com/12273384/108915950-7ee90500-7625-11eb-953e-b1d31aa51121.png)

I'd actually always download the latest version if possible.